pywren/wrenhandler.py

```diff
@@ -225,6 +229,13 @@ def generic_handler(event, context_dict, custom_handler_env=None):
         status_key = event['status_key']
         func_key = event['func_key']
         data_key = event['data_key']
+        cancel_key = event['cancel_key']
+
+        # Check for cancel
+        if key_exists(s3_client, s3_bucket, cancel_key):
+            raise Exception("Function cancelled")
```

Comment:
shouldn't this be `Exception("CANCELLED", "Function cancelled")` ?

Reply:
You're correct, will fix this